### PR TITLE
FixCode: add a fixit to insert the missing enum element cases in switch statements.

### PR DESCRIPTION
**Pull request by nkcsgexi**
This doesn't work on entirely empty switch block because we reject such statement during parsing.

### CODE REVIEW

**Comment by nkcsgexi** [comment]
@swift-ci please smoke test

File: lib/SILOptimizer/Mandatory/DataflowDiagnostics.cpp [hunk 1]
```diff
@@ -57,6 +60,81 @@ static void diagnoseMissingReturn(const UnreachableInst *UI,
...
+  } else {
+    return true;
+  }
+  return false;
+};
+
+static void diagnoseMissingCases(ASTContext &Context,
+                                 const SwitchStmt *SwitchS) {
+  SourceLoc EndLoc = SwitchS->getEndLoc();
+  StringRef PlaceHolder = "<#Code#>";
```
Comment (jrose-apple) [inline lib/SILOptimizer/Mandatory/DataflowDiagnostics.cpp]: Tiny nitpick: "placeholder" is one word.

**Comment by nkcsgexi** [comment]
@swift-ci please smoke test

**Comment by nkcsgexi** [comment]
@swift-ci please smoke test

**Pull request merged**